extensions/amp-story/0.1/bookend-share.js

```diff
@@ -0,0 +1,179 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {ICONS} from './icons';
+import {Services} from '../../../src/services';
+import {isObject} from '../../../src/types';
+import {createElementWithAttributes, escapeHtml} from '../../../src/dom';
+import {dev, user} from '../../../src/log';
+import {dict} from './../../../src/utils/object';
+
+
+const LINK_SHARE_TEMPLATE =
+    `<div class="i-amphtml-story-share-icon">
```

Comment:
We don't allow XSS-able inner HTMLs. Please just use the DOM API.

Reply:
Done